wsproto/connection.py

```diff
@@ -393,6 +393,34 @@ def _process_connection_request(self, event):
 
         return ConnectionRequested(proposed_subprotocols, event)
 
+    def extension_accept(self, extensions_header):
```

Comment:
Should this be public API?
It doesn't really do anything with the connection, and it is only called via `accept`.